doc/quota.md

```diff
@@ -14,14 +14,44 @@ metadata:
     app: containerized-data-importer
     cdi.kubevirt.io: ""
   name: config
-spec: {
+spec:
     podResourceRequirements:
-    limits:
-      cpu: "4"
-      memory: "1Gi"
-    requests:
-      cpu: "1"
-      memory: "250Mi"
-}
+      limits:
+        cpu: "4"
+        memory: "1Gi"
+      requests:
+        cpu: "1"
+        memory: "250Mi"
+
 ```
+
 Once the CDIConfig object is updated, the status section of the object will reflect that values that will be used to pass to the pods. [limits and requests](https://kubernetes.io/docs/tasks/administer-cluster/manage-resources/memory-default-namespace/#motivation-for-default-memory-limits-and-requests) are explained in the kubernetes documentation.
```

Comment:
This is an old doc, thanks for helping to make it current.  I think it would be worth mentioning that the CDIConfig singleton resource is read only and it is updated via `spec.config` section of the CDI custom resource.

Once the CDI resource is updated, CDI config `status.defaultPodResourceRequirements` should reflect the changes

Reply:
I'm not saying I disagree, though I'm not sure I'd word it exactly like that and I'd prefer not to guess.  If you want to add the changes to this PR, put it in a comment and I'll add it, if it's a blocker, not a blocker I'm indifferent.